release/release_data_tests.yaml

```diff
@@ -379,6 +379,7 @@
       --data-directory 300G-image-data-synthetic-raw-parquet --data-format parquet
 
   variations:
+    - __suffix__: regular
```

Comment:
should we name it `autoscaling_compute` to be consistent with the other variant?

Reply:
Addressed